src/Klarna.Checkout/KlarnaCheckoutService.cs

```diff
@@ -550,21 +547,21 @@ protected virtual IEnumerable<string> GetCountries()
             return CountryCodeHelper.GetTwoLetterCountryCodes(countries.Country.Select(x => x.Code));
         }
 
-        protected virtual ICheckoutOrder CreateCheckoutOrder(IMarket market)
-        {
-            return CreateCheckoutOrder(market, client => client.NewCheckoutOrder());
-        }
+        //protected virtual ICheckoutOrder CreateCheckoutOrder(IMarket market)
```

Comment:
Should this be removed?

Reply:
I think yes, since now it's different how client is created.